ACCESSIBILITY.md

```diff
@@ -0,0 +1,90 @@
+
+While accessibility is commonly understood with respect to disability, we – meaning _The Turing Way_ community – define accessibility as giving access to everyone, with specific attention given to ensuring proper access for groups of people that encounter different barriers for participation. Barriers may be anything related to, including, and extending beyond the following: 
+- Age
+- Disability
+- Ethnicity and race
+- Family
+- Language (written and/or spoken)
+- Geography
+- Nationality
+- Religion, faith and belief
+- Sex and gender
+- Sexual orientation
+- Socioeconomic status
+- Technical or disciplinary background
+- Mental health diagnoses 
+- Low bandwidth internet access or unstable connectivity (i.e. connection via mobile)
+- Using devices with limited computational resources or capacity
+
+We recognise the complex and interlinking ways in which intersectionality affects identity, and thereby, access or inclusion within any community project like _The Turing Way_. So in this sense, our accessibility practices may overlap with practices that enable inclusion more broadly within any community.
+
+In developing this policy, we aim to draw from the [access-centered framework](https://accesscenteredmovement.com/what-access-centered-means/) in describing access as a constant process that changes in each space with each individual. This document is a part of a broader collection of documentation across accessibility, listed below:
+- [Accessibility policy](https://github.com/the-turing-way/the-turing-way/blob/main/ACCESSIBILITY.md): Our vision and guiding principles for how we practice accessibility.
+- [Community Handbook Accessibility Guidelines](https://book.the-turing-way.org/community-handbook/accessibility): Practical advice to build accessibility into your contributions and activity within the community
+- Accessibility Guide: A broader discussion of accessibility and access-related practices that extend beyond The Turing Way
+
+Each resource (our Accessibility Policy - this document, Community Handbook Accessibility Guidelines, and our Accessibility Guide) play a different yet overlapping role in enabling, documenting, and sharing our evolving language around accessible and inclusive collaboration.
+
+This policy is a work in progress, as are the practices that we employ as a community. Much like how _The Turing Way_ itself is always being written and rewritten, our accessibility policies and practices are constantly being updated as broader awareness grows and changes in technology are made.
+
+## 1.1 Core Principles
+
+These core principles are at the heart of our accessibility policy, and they extend beyond the web accessibility of our open access guides to the broader practices that we use as a community, expanded upon in Section 2.0.
```

Comment:
Suggested rephrasing:

'...to our broader practices as a community, expanded upon in Section 2.0'